js/iframe-coordinator/karma.conf.js

```diff
@@ -0,0 +1,26 @@
+let webpackConfig = require('./webpack.config');
+
+module.exports = function(config) {
+  config.set({
+    basePath: '',
+    frameworks: ['jasmine'],
+    files: [
+      '**/*.spec.ts'
+    ],
+    exclude: [
+      'node_modules/**/*.spec.ts'
+    ],
+    preprocessors: {
+      '**/*.spec.ts': ['webpack']
+    },
+    webpack: webpackConfig,
+    reporters: ['progress'],
+    port: 9876,
+    colors: true,
+    logLevel: config.LOG_INFO,
+    autoWatch: true,
+    browsers: ['Chrome'],
```

Comment:
I am not sure how testing is normally done with our builds.  If we can use chrome, or if we have to use headless.  Let me know whats the best practice.

Reply:
We usually install chromium-headless for CI runs.

In some of our projects, we use qunit which differentiates between desktop and ci runs.  It uses Chromium/Chrome local install in the former and then chromium headless in the ci runs.

Although the index.js file is specifically for Ember/QUnit, you may be able to just add this library as a dependency and use some of the scripts to check/install.

https://github.com/purecloudlabs/ember-chromium

Reply:
We'll definitely need a headless option for CI builds.

Reply:
Alright, I'll set get that up here on Monday.